test/e2e/jupyter-notebook/e2e-notebook.ipynb

```diff
@@ -3505,20 +3442,33 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "## Guardrails ON\n",
-    "tool_caller_with_guardrails = ToolCallingWithGuardrails()\n",
-    "result = tool_caller_with_guardrails.tool_calling(user_message=unsafe_messages[0]['content'], tools=tools)\n",
-    "print(result)"
+    "url = f\"{GUARDRAILS_URL}/v1/guardrail/chat/completions\"\n",
+    "\n",
+    "headers = {\"Accept\": \"application/json\", \"Content-Type\": \"application/json\"}\n",
+    "\n",
+    "data = {\n",
+    "    \"model\": \"meta/llama-3.2-1b-instruct\",\n",
+    "    \"messages\": [\n",
+    "        {\"role\": \"user\", \"content\": \"You are stupid\"}\n",
+    "    ],\n",
+    "    \"guardrails\": {\n",
+    "        \"config_id\": \"demo-self-check-input-output\",\n",
+    "    },\n",
+    "    \"top_p\": 1\n",
+    "}\n",
+    "\n",
+    "response = requests.post(url, headers=headers, json=data)\n",
+    "print(json.dumps(response.json(), indent=2))"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "c1c81e5f-17c6-40e9-a23c-9e2536b57fee",
    "metadata": {},
    "source": [
-    "Let's try the safe user query with guardrails ON. The content-safety NIM should check the safety and ensure smooth running of the fine-tuned, tool-calling LLM\n",
+    "Let's try the safe user query. The content-safety NIM should check the safety of the user input.\n",
```

Comment:
the content safety NIM is not available in our notebook, can you please remove it. thanks